tests/test_model_framework.py

```diff
@@ -53,15 +53,21 @@ class TestModelFrameworkMST(SMRFTestCase):
     """
     Test timezone handling for MST.
     """
-    TIMEZONE = pytz.timezone('US/Mountain')
+    TIMEZONE = pytz.timezone('MST')
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
         base_config = copy.deepcopy(cls.base_config)
-        base_config.cfg['time']['time_zone'] = str(cls.TIMEZONE)
+        base_config.cfg['time']['time_zone'] = 'MST'
```

Comment:
Did this call need to b changed?
Or did you want to create a more 'real world' input scenario with it?

Reply:
It's a more real world application as we don't typically pass a pytz object. This will also test that the inicheck parser is working. Then `TIMEZONE` becomes what is checked against